astropy/utils/introspection.py

```diff
@@ -123,7 +124,10 @@ def minversion(module, version, inclusive=True, version_path='__version__'):
     """
     # import LooseVersion here to avoid conflicts between setuptools and
     # distutils. See https://github.com/astropy/astropy/pull/10571
-    from distutils.version import LooseVersion
+    with warnings.catch_warnings():
+        # distutils package is deprecated in Python 3.10
+        warnings.simplefilter('ignore', DeprecationWarning)
+        from distutils.version import LooseVersion
```

Comment:
Oh no... How do we replace this?

Reply:
Yes, it seems time to get back to that one!!